newrelic/resource_newrelic_synthetics_script_monitor.go

```diff
@@ -73,6 +73,16 @@ func syntheticsScriptBrowserMonitorAdvancedOptionsSchema() map[string]*schema.Sc
 			Optional:    true,
 			Description: "Capture a screenshot during job execution.",
 		},
+		"device_orientation": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "The device orientation the user would like to represent. Valid values are LANDSCAPE, PORTRAIT, or NONE.",
```

Comment:
Any thoughts on having ValidateFunc here?

Reply:
I'd rather let the API handle the error scenario. I will add a test for that as well.